### DISCUSSION TITLE
Panlee zx3d50ce08s v15 usrc wireless-tag

### DISCUSSION THREAD

**Discussion by sammcj** [HASwitchPlate/openHASP#508] [Hardware]
Howdy, I'm just getting started with openHASP and picked myself up a Panlee zx3d50ce08s v15 usrc.

I'm just wondering if there's any special firmware I should be using with it, or if the `wt32-sc01-plus_full_16MB_v0.7.0-rc6_2f4092b.bin` is fine?

![planlee-zx3d50ce08s-v15-usrc](https://github.com/HASwitchPlate/openHASP/assets/862951/6439c5da-9482-4fc8-acf7-ce2de5ea406f)

**Comment by fvanroie** [comment]
Panlee zx3d50ce08s v15 usrc is the PCB made by Panlee that is used in the WT32-SC01 Plus device. Essentially they are the same pinouts so the `wt32-sc01-plus_full_16MB_v0.7.0-rc6_2f4092b.bin` will work fine.

**Reply by sammcj** [reply]
Thanks! I managed to get it up and going without much trouble.

Now to try and figure out how to wire a battery up to it!
